### PR TITLE
Style Searchbar

### PR DESCRIPTION
**Pull request by patilnandini1811**
- Provide a maximum width for larger screens.
- Introduce a border with a red hue.
- Add a smooth transition effect for the box-shadow on focus.

### CODE REVIEW

File: src/components/Searchbar/Searchbar.jsx [hunk 1]
```diff
@@ -12,6 +13,7 @@ const SearchBar = ({ setSearchQuery }) => {
         type="text"
         placeholder="Search..."
         onChange={handleInputChange}
+        className={styles.netflixSearchBar}
```
Comment (SabanaSK) [inline src/components/Searchbar/Searchbar.jsx]: Stylelint prefer kebab case

File: src/components/Searchbar/Searchbar.module.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  border-radius: 50px;
+  outline: none;
+  transition: box-shadow 0.3s ease;
+
+  ::placeholder {
+    color: rgba(255, 255, 255, 0.5);
+  }
+
+  :focus {
+    box-shadow: 0 0 10px rgba(255, 0, 0, 0.5);
```
Comment (SabanaSK) [inline src/components/Searchbar/Searchbar.module.css]: I have create color shade in :root at App.css, you can reuse that instead of creating new colors

Comment (SabanaSK) [reply src/components/Searchbar/Searchbar.module.css]: If the colors is new colors then add to the root

Comment (patilnandini1811) [reply src/components/Searchbar/Searchbar.module.css]: ok

**Review by SabanaSK — CHANGES_REQUESTED** [review]
The design look good, just some small change

File: src/components/Searchbar/Searchbar.module.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  }
+
+  @media (max-width: 768px) {
+    font-size: 14px;
+    padding: 8px 16px;
+  }
+
+  @media (max-width: 480px) {
```
Comment (SabanaSK) [inline src/components/Searchbar/Searchbar.module.css]: Here you also get warning for fix media-feature-name-value-allowed-list from stylelint

Comment (SabanaSK) [reply src/components/Searchbar/Searchbar.module.css]: I recommend you to run stylelint before pushing : )

File: src/components/Searchbar/Searchbar.jsx [hunk 4]
```diff
@@ -12,6 +13,7 @@ const SearchBar = ({ setSearchQuery }) => {
         type="text"
         placeholder="Search..."
         onChange={handleInputChange}
+        className={styles[ 'search-bar' ]}
```
Comment (SabanaSK) [inline src/components/Searchbar/Searchbar.jsx]: One last small change, use " instead of '

File: src/components/Searchbar/Searchbar.jsx [hunk 5]
```diff
@@ -1,4 +1,5 @@
 import PropTypes from 'prop-types';
```
Comment (SabanaSK) [inline src/components/Searchbar/Searchbar.jsx]: "prop-types" instead of 'prop-types'

**Review by SabanaSK — APPROVED** [review]

**Review by SabanaSK — APPROVED** [review]
awesome

**Pull request merged**